openmc/examples.py

```diff
@@ -0,0 +1,619 @@
+    l201.pitch = (21.42, 21.42)
+    l201.universes = [
+        [fa_hw]*21,
+        [fa_hw]*21,
+        [fa_hw]*7 + [fa_hot]*7 + [fa_hw]*7,
+        [fa_hw]*5 + [fa_hot]*11 + [fa_hw]*5,
+        [fa_hw]*4 + [fa_hot]*13 + [fa_hw]*4,
+        [fa_hw]*3 + [fa_hot]*15 + [fa_hw]*3,
+        [fa_hw]*3 + [fa_hot]*15 + [fa_hw]*3,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*2 + [fa_hot]*17 + [fa_hw]*2,
+        [fa_hw]*3 + [fa_hot]*15 + [fa_hw]*3,
+        [fa_hw]*3 + [fa_hot]*15 + [fa_hw]*3,
+        [fa_hw]*4 + [fa_hot]*13 + [fa_hw]*4,
+        [fa_hw]*5 + [fa_hot]*11 + [fa_hw]*5,
+        [fa_hw]*7 + [fa_hot]*7 + [fa_hw]*7,
+        [fa_hw]*21,
+        [fa_hw]*21]
+
+    # Define root universe.
+    root = openmc.Universe(universe_id=0, name='root universe')
+    c1 = openmc.Cell(cell_id=1, fill=l200, region=-s6 & +s34 & -s35)
+    c2 = openmc.Cell(cell_id=2, fill=l201, region=-s6 & +s35 & -s36)
+    c3 = openmc.Cell(cell_id=3, fill=bot_plate, region=-s7 & +s31 & -s32)
+    c4 = openmc.Cell(cell_id=4, fill=bot_nozzle, region=-s5 & +s32 & -s33)
+    c5 = openmc.Cell(cell_id=5, fill=bot_fa, region=-s5 & +s33 & -s34)
+    c6 = openmc.Cell(cell_id=6, fill=top_fa, region=-s5 & +s36 & -s37)
+    c7 = openmc.Cell(cell_id=7, fill=top_nozzle, region=-s5 & +s37 & -s38)
+    c8 = openmc.Cell(cell_id=8, fill=upper_rad_ref, region=-s7 & +s38 & -s39)
+    c9 = openmc.Cell(cell_id=9, fill=bot_nozzle, region=+s6 & -s7 & +s32 & -s38)
+    c10 = openmc.Cell(cell_id=10, fill=rpv_steel, region=+s7 & -s8 & +s31 & -s39)
+    c11 = openmc.Cell(cell_id=11, fill=lower_rad_ref, region=+s5 & -s6 & +s32 & -s34)
+    c12 = openmc.Cell(cell_id=12, fill=upper_rad_ref, region=+s5 & -s6 & +s36 & -s38)
+    root.add_cells((c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12))
+
+    # Assign root universe to geometry
+    model.geometry.root_universe = root
+
+    model.settings.batches = 10
+    model.settings.inactive = 5
+    model.settings.particles = 100
+    model.settings.source = openmc.Source(space=openmc.stats.Box(
+        [-160, -160, -183], [160, 160, 183]))
+
+    plot = openmc.Plot()
+    plot.origin = (125, 125, 0)
+    plot.width = (250, 250)
+    plot.pixels = (3000, 3000)
+    plot.color_by = 'material'
+    model.plots.append(plot)
+
+    return model
+
+
+def pwr_assembly():
+    """Create a PWR assembly model.
+
+    Returns
+    -------
+    model : openmc.model.Model
+        A PWR assembly model
+
+    """
+
+    model = openmc.model.Model()
+
+    # Define materials.
+    fuel = openmc.Material(name='Fuel')
+    fuel.set_density('g/cm3', 10.29769)
+    fuel.add_nuclide("U234", 4.4843e-6)
+    fuel.add_nuclide("U235", 5.5815e-4)
+    fuel.add_nuclide("U238", 2.2408e-2)
+    fuel.add_nuclide("O16", 4.5829e-2)
```

Comment:
Perhaps it's because many of us don't even know it exists, or that we have to manually turn it on ;-) At least that was the case for my work, though it didn't matter for me anyway.